client/src/layouts/Header.vue

```diff
@@ -148,7 +148,12 @@ export default {
       this.searchPostByEmogi()
     },
     searchPostByEmogi () {
-      this.$router.push(`/?tab=search&emoji=${this.userInput}`)
+      if (this.$route.query.emoji === this.userInput) {
+        this.$router.push('')
```

Comment:
Ok this is sad one;(
I'd suggest finding better solution after demo, or even moving search from route to state.